### DISCUSSION TITLE
Get Radio Channels to the audioMotion player, Hack the Planet!

### DISCUSSION THREAD

**Discussion by JesusGod-Pope666-Info** [hvianna/audioMotion-analyzer#57] [General]
Turn on VPN in Opera Settings. push the VPN and set it to America. Check if the Radio station now plays, if so - Right click the page and inspect element, network, media, Control R - enjoy :)

![Screenshot_2023-09-24_09-36-21](https://github.com/hvianna/audioMotion-analyzer/assets/90806094/5a24fa83-3086-4124-b5c8-29444694f023)

Not sure if the Radio stations changes the URL at anytime but... it works.

Anyone know the m3u code for the 3 line information in the aM player?

Here is a website blocking me, worked like a hot knife through butter with the combined tricks ;) So sickening you need to do it, and many more crazies are out there of utter madness making you do much labor to get things sorted.
![Screenshot_2023-09-23_21-57-27](https://github.com/hvianna/audioMotion-analyzer/assets/90806094/116b980b-c680-4014-88e0-ed8a89d57383)

Sharing is Caring and Copyright is of the Devil and security mostly tyrannic control by others
---- Darkijah

**Comment by JesusGod-Pope666-Info** [comment]
My last one:

Drop us a line at lavada@menorahmusic.com and let us know you are listening!---We offer a gentle mix of Messianic & Hebraic Roots, Country & Southern Gospel, Inspirational Jazz, & Solo Piano Christian Music. We "remember the Sabbath Day" here at MMIR in the Texas Hill Country and offer up some special shabbat music from sundown Friday evening to sundown Saturday evening! We encourage you to visit the websites and support the ministries of our artists by purchasing their albums! Thank you for listening!

- - -

live 365 radio place is censoring other people from other nations. You can circumvent this by the way. And that is what I am going to do now for getting you station on my own player.


https://ais-edge102-live365-dal02.cdnstream.com/a42872?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1695540867189

https://streaming.live365.com/a42872?listenerId=Live365-AdBlock&aw_0_1st.playerId=Live365&aw_0_1st.skey=1695540867189

Not sure which one is the better one here, but both works.

-
Darkijah

Picture was included for help so they can do it themself.

**Comment by JesusGod-Pope666-Info** [comment]
Although one of the things that is of issue, I would think the Programmer could sort out somehow..... Files works in VLC and yet won't work in the player. And then you need to hack the planet again. But I would think the programmer should be able to at least sort that issue out? The other one is harder, with country nonsense....

But whatever, hope this is help full for people who want to add radio channels to the player :) I'll see if i can post my list when done although most of those Christians channels are way to much worldly for my taste and pushing lawlessness.

But have found some Messianic! :) 3 of them at least, maybe 4 with this last one.... but anyway....

Still need to figure out how to get the ALBUM data added I think it is for the m3u file, HELP HELP HELP!!!! I have tried different things like album stuff but nothing has yet worked. We don't want people to have these issues, some examples out of the box should be available for show, just 1 is more then enough.

**Comment by JesusGod-Pope666-Info** [comment]
Add /music folder with one mp3 and 1 radio channel m3u with all 3 details.
/backgrounds folder with 1 image and one 1 video background for show.
and the .htaccess with Options +Indexes at the root level for all things to work out of the box, or if you want in the specific folders - just so it works and makes less hazzle for the newcomer. Less issues, more users, bigger community - more movement. :)

**Comment by JesusGod-Pope666-Info** [comment]
Maybe one should use rather the sentence, hack the ball!!! In these days of flat tards, but I guess a planet is round as well.

Naa anyway.... listening to some Radio channels, and have many to sort out and figure out how to do it. And still missing that 3 line for some kind of information.

**Comment by JesusGod-Pope666-Info** [comment]
non of your m3u examples have that 3 line so I have no idea what the code is to make it work - here you have the first 2:
![Screenshot_2023-09-24_10-13-32](https://github.com/hvianna/audioMotion-analyzer/assets/90806094/46a04c57-0d62-4043-840a-40f0d8ddb9db)

**Comment by JesusGod-Pope666-Info** [comment]
Is there a way to connect pictures to each Radio m4u file? Like folder image... the in the other media app he calls it poster or thumbnail.... added to the image... would be nice if you supported that as well if I get it linked together with the media folder :)

If you want, we could call each other up.... if you wanna have a direct talk on things.

![Screenshot_2023-09-24_10-20-38](https://github.com/hvianna/audioMotion-analyzer/assets/90806094/bf009cb6-8d78-45aa-8ab1-9542a7524d22)

**Comment by JesusGod-Pope666-Info** [comment]
By the way, you clearly "easily" could make this play video files as a player... Like I just realized it already play video files, as to say in the background. You could make this into a free open source player that is very very useful against the powers to be for anyone who wanna skip the censorship of many platforms. You can still see much of the layout I did in the other player I payed and bought - but your whole thing has a LOT of potential to even become better. We don't need to focus on all the add nonsense stuff and bloated features, just have it work as sane as in the old days of computing! :)

Lightweight and direct and mobile functionality and your project could become a giant to use.

**Reply by JesusGod-Pope666-Info** [reply]
If you want any ideas then check my player and the layout - at least some kind of idea.... We could keep the folder idea to the left bar as playlists. And then make it work out so the user don't need to know much coding either yet be able to do stuff, adding picture and text details by folder connection to files which pretty much would be the first go media for people :)

There are some great possibilities here....

**Reply by JesusGod-Pope666-Info** [reply]
And it runs in mobile as well, although... have not been easy and you need to flip it 90 degree, but that seems to work pretty good, the best that I could do overall: https://jesusgod-pope666.info/uvpf.php#/?playlistId=3&videoId=0

But we could make a free open source platform and even make it better, no nonsense bloated and running smooth being lightweight. Again you pretty much have a lot already working - "just" need some design fixing and some added stuff here there and you have a powerful Media Player open source and free. :)

**Reply by hvianna** [reply]
Yeah, there's a lot to be done for sure. I just wish I had more free time :)

I'd appreciate it if you could organize all your suggestions in single concise posts in the [Ideas](https://github.com/hvianna/audioMotion.js/discussions/categories/ideas) section of the forum.

**Comment by JesusGod-Pope666-Info** [comment]
Seems... when you download those m4u files which does not work: http://192.241.185.108:8013/listen.pls?sid=1

You just need to delete the end and insert whatever like http://192.241.185.108:8013/JesusGod-Pope666.Info

Here is another: http://s2.voscast.com:8796/listen.pls?sid=1 and just doing, http://s2.voscast.com:8796/JesusGod-Pope666.Info should make it work. Weird.... oddly enough it does not work if you don't write anything after the / so you have to write something....

**Comment by JesusGod-Pope666-Info** [comment]
Or if there is nothing then just add whatever after the / like this one was empty: http://199.180.72.2:8116/ should now work with: http://199.180.72.2:8116/JesusGod-Pope666.Info

**Comment by JesusGod-Pope666-Info** [comment]
hmmm 4 out of 19 does not work... but overall that is pretty good as a fix.... Lets look into those last ones.
